packages/sol-cov/src/source_maps.ts

```diff
@@ -12,6 +12,9 @@ export interface SourceLocation {
 }
 
 export function getLocationByOffset(str: string): LocationByOffset {
+    if (_.isUndefined(str)) {
```

Comment:
Can we check that above and just not call this function?
I'd prefer that solution because now we're checking for undefined the variable that's guaranteed to be undefined by the type system. Let's either change the types or check before.